cartoframes/data/clients/bigquery_client.py

```diff
@@ -0,0 +1,36 @@
+from ..enrichment import fake_auth
+
+from google.cloud import bigquery
+
+# TODO: decorator to authenticate
+
+
+class BigQueryClient(object):
+
+    def __init__(self, credentials):
+        token = fake_auth.auth(credentials)
+
+        self.credentials = credentials
+        self.client = bigquery.Client().from_service_account_json(token)  # Change auth method when token received
+
+    def upload_dataframe(self, dataframe, schema, tablename, project, dataset):
+        dataset_ref = self.client.dataset(dataset, project=project)
+        table_ref = dataset_ref.table(tablename)
+
+        schema_wrapped = [bigquery.SchemaField(column, dtype) for column, dtype in schema.items()]
+
+        job_config = bigquery.LoadJobConfig()
+        job_config.schema = schema_wrapped
+
+        job = self.client.load_table_from_dataframe(dataframe, table_ref, job_config=job_config)
+        job.result()
+
+    def query(self, query, **kwargs):
+        response = self.client.query(query, **kwargs)
+
+        return response
+
+    def delete_table(self, tablename, project, dataset):
```

Comment:
We have not decided yet how to delete tables, but at the moment the role we'll use in BQ does not have delete permission.

That means when we integrate with the `token` API, we might have to disable deletion of tables.

Reply:
:+1:

Reply:
I will add TTL when uploading table and avoid deleting at the end of the process

Reply:
No needed, we want to deal with this in a separate issue. Don't worry about that by now.